operations/helm/charts/mimir-distributed/ci/offline/metamonitoring-values.yaml

```diff
@@ -7,3 +7,22 @@ metaMonitoring:
   grafanaAgent:
     enabled: true
     installOperator: true
+    metrics:
+      # Leaving the remote empty should use the default to send it to Mimir directly
```

Comment:
```suggestion
      # Leave the remote empty to use the default to send it to Mimir directly
```

Reply:
Rationale: "Should" is ambiguous. Does it mean that it will or will not use the default?

Reply:
makes sense

btw thanks for reviewing these even though I didn't put an explicit docs label 🙂